include/zendoo_mc.h

```diff
@@ -6,7 +6,10 @@
 
 static const size_t SC_PROOF_SIZE = 771;
 static const size_t SC_VK_SIZE = 1544;
-static const size_t SC_FIELD_SIZE = 96;
+static const size_t SC_FIELD_SIZE = 32;
```

Comment:
It would be better to use some getters on the Rust side to retrieve these values from the related ojects to avoid unalignment between these hardcoded parameters and their actual values

Reply:
I added the proposed getters, the alignment will be ensured in a cpp piece of code which asserts their consistency on MC node startup.